README.md

```diff
@@ -213,25 +220,27 @@ a [Kafka Connect wrapper](https://github.com/conduitio/conduit-kafka-connect-wra
 that allows you to run any Apache Kafka Connect connector as part of a Conduit
 pipeline.
 
-If you are interested in writing a connector yourself, have a look at our
-[Go Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk). Since
-standalone connectors communicate with Conduit through gRPC they can be written
-in virtually any programming language, as long as the connector follows
-the [Conduit Connector Protocol](https://github.com/ConduitIO/conduit-connector-protocol)
-.
+If you are interested in writing a connector yourself, have a look at
+our [Go Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk).
+Since standalone connectors communicate with Conduit through gRPC they can be
+written in virtually any programming language, as long as the connector follows
+the [Conduit Connector Protocol](https://github.com/ConduitIO/conduit-connector-protocol).
 
 ## Processors
 
 A processor is a component that operates on a single record that flows through a
 pipeline. It can either change the record (i.e. **transform** it) or **filter**
 it out based on some criteria.
 
-Conduit provides a number of built-in processors, which can be used to manipulate fields,
-send requests to HTTP endpoints, and more, check [Builtin processors](https://conduit.io/docs/using/processors/builtin/)
+Conduit provides a number of built-in processors, which can be used to
+manipulate fields, send requests to HTTP endpoints, and more,
+check [Builtin processors](https://conduit.io/docs/using/processors/builtin/)
```

Comment:
```suggestion
check [built-in processors](https://conduit.io/docs/using/processors/builtin/)
```

P.S. I know that we use "builtin" on the site we link to, but I think "[built-in](https://ell.stackexchange.com/a/16887)" is the way to go.